ghmirror/core/mirror_requests.py

```diff
@@ -210,16 +180,16 @@ def online_request(method, url, auth, data=None, url_params=None):
     )
 
     if resp.status_code == 304:
-        return _handle_not_changed(
```

Comment:
I think I still do not fully grasp this. Are there other cases of 304, where we'd still have to get the page? Like changed content?

Reply:
No, a 304 response means no changes in the content. This method was just really updating the Link header with the new value, but the content was not getting modified. 
I paste here the same example explained in the ticket. As you can see, the first page is a HIT (ETAG and last-modified had not changed) but the pagination data in the header returned by the Github API has one more page. 

```
curl -o /dev/null --header "Authorization: Bearer FOO" --user-agent "PyGithub/Python" https://github-mirror.devshift.net/organizations/43133889/team/4009942/members -v 2>&1 | grep -e x-cache -e link:
< x-cache: ONLINE_HIT
< link: <https://github-mirror.devshift.net/organizations/43133889/team/4009942/members?per_page=30&page=2>; rel="next", <https://github-mirror.devshift.net/organizations/43133889/team/4009942/members?per_page=30&page=7>; rel="last"

curl -o /dev/null --header "Authorization: Bearer FOO" --user-agent "PyGithub/Python" https://api.github.com/organizations/43133889/team/4009942/members -v 2>&1 | grep -e link:
< link: <https://api.github.com/organizations/43133889/team/4009942/members?per_page=30&page=2>; rel="next", <https://api.github.com/organizations/43133889/team/4009942/members?per_page=30&page=8>; rel="last"
```